src/Propagation/BinaryFormat.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Propagation;
+
+use OpenTelemetry\Context\SpanContext;
+
+final class BinaryFormat
```

Comment:
Does the spec expect us to provide an implementation of the binary format? If so it's odd that there isn't a description or spec for the format. If it doesn't expect us to provide an impl, then this should probably just be an interface, I think?

Reply:
The binary format of the spec is pretty ambiguous now:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-propagators.md#binary-format

I'd expect that might change in the future?

Reply:
I agree, it is a bit odd that it's not in the spec. I based what is done here on what is used for the Go library as I said in my initial message but if you guys want to be a bit more cautious, I could swap it with a simple interface for now.